hw/ip_templates/rv_core_ibex/rtl/rv_core_ibex.sv.tpl

```diff
@@ -579,6 +580,25 @@ module ${module_instance_name}
     end
   end
 
+  logic instr_req_q;
+  logic [31:0] instr_addr_q;
+  // Add optional pipeline stage between the Ibex and the address translation
```

Comment:
```suggestion
  // Add an optional pipeline stage between Ibex and the address translation
```

Small nitpick. It sounds more natural to me without the article. So the same way you wrote it in the hjson description.